conjureplugin/config/config.go

```diff
@@ -61,30 +76,41 @@ func (c *ConjurePluginConfig) ToParams(stdout io.Writer) (conjureplugin.ConjureP
 		// if value for "publish" is not specified, treat as "true" only if provider generates IR from YAML
 		if currConfig.Publish == nil {
 			publishVal = irProvider.GeneratedFromYAML()
+		} else {
+			publishVal = *currConfig.Publish
 		}
 		acceptFuncsFlag := true
 		if currConfig.AcceptFuncs != nil {
 			acceptFuncsFlag = *currConfig.AcceptFuncs
 		}
 		params[key] = conjureplugin.ConjureProjectParam{
-			OutputDir:   currConfig.OutputDir,
-			IRProvider:  irProvider,
-			AcceptFuncs: acceptFuncsFlag,
-			Server:      currConfig.Server,
-			CLI:         currConfig.CLI,
-			Publish:     publishVal,
-			GroupID:     groupID,
+			OutputDir:                outputDir,
+			IRProvider:               irProvider,
+			AcceptFuncs:              acceptFuncsFlag,
+			Server:                   currConfig.Server,
+			CLI:                      currConfig.CLI,
+			Publish:                  publishVal,
+			GroupID:                  groupID,
+			SkipDeleteGeneratedFiles: currConfig.SkipDeleteGeneratedFiles,
```

Comment:
If `SkipDeleteGeneratedFiles` is `true`, then we will probably want some form of best-effort verification that there are no output directories that are subdirectories of another (not just normalized match), since otherwise deleting all generated files within a project could still cause issues with deleting files generated by other projects.